pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -80,6 +80,12 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 	if err := ctrl.cleanUpDuplicatedMC(managedNodeConfigKeyPrefix); err != nil {
 		return err
 	}
+	// explicitly setting the cgroupMode to "v1"
+	// (TODO) This code can be remove this code in future releases
```

Comment:
nit: technically this code is only in 4.13. Maybe change this comment to:

`This code only exists in 4.13 to set the default to v1, if the user did not otherwise provide it. All clusters must upgrade through this 4.13.z before upgrade to 4.14 to preserve v1 defaults`

Something along those lines?

Reply:
Sure, Thanks !!